pkg/releasenotes/print.go

```diff
@@ -16,13 +16,17 @@ const templ = `## v{{ .Milestone }}
 
 Released on {{ .Day }}
 
+{{ if .BreakingNotes }}
+### Breaking Changes :warning:
 
-### Major Changes
-
-{{ if .MajorNotes }}
 {{ range .BreakingNotes }}
 * {{ .Description }} [[#{{ .Num }}]({{ .URI }})] - [{{ .Author }}]({{ .AuthorURL }})
 {{ end }}
+{{ end }}
+
+{{ if .MajorNotes }}
```

Comment:
This way, it is similar to what we do for other sections. Avoids some clutters in patch releases ;)